index.ts

```diff
@@ -93,7 +94,7 @@ function delegate<
 	selector: Selector,
 	type: TEventType,
 	callback: delegate.EventHandler<GlobalEventHandlersEventMap[TEventType], TElement>,
-	options?: boolean | AddEventListenerOptions
+	options?: boolean | Except<AddEventListenerOptions, 'once'>
```

Comment:
I think we can skip the extra dependency here, especially because this is not a devDependency. This type appears in the generated `index.d.ts` file and will fail when imported by third parties.

```suggestion
	options?: boolean | Omit<AddEventListenerOptions, 'once'>
```

`Omit` is only bad because you can use it as `Omit<OtherType, 'mistyped-ooonce'>` and it doesn't fail. This isn't really a problem here.

Reply:
Fixing…